src/network/network_connection.cpp

```diff
@@ -73,20 +88,28 @@ void NetworkConnection::Init(short event_flags, NetworkThread *thread,
 
   event_add(network_event, nullptr);
   event_add(workpool_event, nullptr);
+  event_add(logpool_event, nullptr);
```

Comment:
I still doubt why we need this logpool_event. Can't we just use the workerpool_event? Since it would be no way for worker in worker pool and logger to invoke the event simultainously.

Reply:
I don't like the idea of having two different events with two different purposes sharing the same trigger. They should do the same thing (trigger the state machine) but I don't think we should mix those.